grammar.js

```diff
@@ -560,11 +580,33 @@ module.exports = grammar({
 
     finally_clause: $ => prec.right(seq('finally', $.expression)),
 
+    binding: $ => prec.dynamic(PREC.binding, seq(
+        field('name', $.identifier),
+        optional(seq(':', field('type', $._param_type))),
+    )),
+
+    bindings: $ => seq(
+        '(',
+        commaSep($.binding),
+        ')',
```

Comment:
Minor nitpick - Could you make the indentation for the new lines consistent with the rest of the file?